andela/ah-frontend-realers
# 167506390 logout :user should be able to log out of the system

#### What . does this PR ?
User is able to log out of the system.

#### Description of the task?
Log out of the system.

#### How can this be manually tested?
After you login click on the icon on your left then you will see a dropdown which has a logout option  then click on it you will be able to logout.
#### What is the relevant pitvotal Tracker stroy
pivotal tracker -Id [167506390](https://www.pivotaltracker.com/n/projects/2350340)
